cmd/dvotenode/dvotenode.go

```diff
@@ -126,6 +127,8 @@ func newConfig() (*config.DvoteCfg, config.Error) {
 		"external address:port to announce to other peers (automatically guessed if empty)")
 	globalCfg.VochainConfig.RPCListen = *flag.String("vochainRPCListen", "127.0.0.1:26657",
 		"rpc host and port to listen to for the voting chain")
+	globalCfg.VochainConfig.DBbackend = db.StorageType(*flag.Int("dbBackend", 0,
+		"database backend for storing the vochain state (0:pebble 1:badger2 2:badger3"))
```

Comment:
how about a string with those three options? much more usable and readable IMO :)